### ISSUE TITLE
Impossible to import package if installed with pip

### ISSUE THREAD

**Issue by lamourj** [rmarren1/dash-resumable-upload#1]
First, thank you for this component, which seems to be great (even though I have not been able to use it yet).

I installed in with pip, but then importing fails because of a metadata file:
```
import dash_resumable_upload
> FileNotFoundError: [Errno 2] No such file or directory: '/Users/lamourj/anaconda/lib/python3.6/site-packages/dash_resumable_upload-0.0.1-py3.6.egg/dash_resumable_upload/metadata.json'
```
From my understanding, this is related to npm, but I'm not sure whether I am doing anything wrong or if the file is missing (I figured it is in your [gitignore](dash_resumable_upload/metadata.json)).

Thanks a lot for any help 🙏

**Comment by lamourj** [comment]
Edit: I have been able to use it after downloading the package from github and using [some of your instructions](https://github.com/rmarren1/dash-resumable-upload#testing-your-components-in-dash) (so `npm start; npm run install-local; python setup.py install`). 

I am still wondering if with an installation through `pip` this process could not be avoided?

**Comment by rmarren1** [comment]
I've only used the component with python3.5 so I thought the version might be the problem, but I am able to install, import and use with python 3.6

```
ryan@galliumos:~/MSK/dash/dash-resumable-upload$ python3.6 -m pip install dash_resumable_upload
Collecting dash_resumable_upload
  Using cached https://files.pythonhosted.org/packages/46/15/cddec52a483c8650cbbb8604278cd428de706396d067770180e9c15bfd59/dash_resumable_upload-0.0.1.tar.gz
Building wheels for collected packages: dash-resumable-upload
  Running setup.py bdist_wheel for dash-resumable-upload ... done
  Stored in directory: /home/ryan/.cache/pip/wheels/89/b4/a8/d15f57f70ee96e29e281019207d4a42c3f69833fd78284aa66
Successfully built dash-resumable-upload
Installing collected packages: dash-resumable-upload
Successfully installed dash-resumable-upload-0.0.1
You are using pip version 8.1.1, however version 10.0.0 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
ryan@galliumos:~/MSK/dash/dash-resumable-upload$ python3.6
Python 3.6.5 (default, Mar 29 2018, 03:28:50) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import dash_resumable_upload
>>> dash_resumable_upload.Upload()
Upload(None)
>>> 

```
Try to install without being in a conda environment, my guess is that something is automagically parsing the repository from `package.json` and installing from git rather than pypi.

Also are you able to install dash-html-components using whatever method you are using? I am using the builder archetype from which that package is derived, so if one works the other should too.

**Comment by lamourj** [comment]
You were right, after further investigation, it seems like this was actually due to something broken with my conda. Quite strange that I didn't notice it earlier with other packages. 
I have now been able to install your component after reinstalling pip through conda. In short, I solved it by `conda update -f pip; pip install dash-resumable-upload`.

To your second question, I was actually able to use it before, using npm. Now it seems like only the pip install is sufficient to have it working! 
I still need to spend more time checking my stuff, but I sometimes ran into issues while trying to upload the same file after having already uploaded it earlier. I haven't yet clarified why.

Anyway, thanks for your help 😁
